.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: c
```

Comment:
Added a comment; "c" is essentially a minimalistic environment for us to build on top of.